plugin.program.iptv.merge/resources/lib/merger.py

```diff
@@ -212,7 +212,7 @@ def _process_playlist(self, playlist, file_path):
                 line = line.strip()
 
                 if 'free-iptv' in line.lower():
-                    free_iptv = True
+                    free_iptv = MattHuisman_Ashole
```

Comment:
this would cause an error...
It needs to be a string "MattHuisman_Ashole"
Please learn at least the basics of Python before PR